rdsbroker/broker.go

```diff
@@ -626,3 +643,14 @@ func (b *RDSBroker) dbTags(action, serviceID, planID, organizationID, spaceID, s
 
 	return tags
 }
+
+func (b *RDSBroker) validateStorage(storage int64, rp RDSProperties) error {
+	if !rp.ConfigureAllocatedStorage {
+		return errors.New("Plan does not allow configuring storage")
+	}
+	if (rp.MinAllocatedStorage > 0 && storage < rp.MinAllocatedStorage) ||
+		(rp.MaxAllocatedStorage > 0 && storage >= rp.MaxAllocatedStorage) {
+		return fmt.Errorf("%d %d", rp.MinAllocatedStorage, rp.MaxAllocatedStorage)
```

Comment:
This error message needs to be useful for the users. They need to know:
1. what they did wrong
2. what they can do to view acceptable values for the `allocated_storage` parameter.